.gitignore

```diff
@@ -64,3 +64,5 @@ typings/
 .idea/**
 jmeter/out
 
+package-lock.json
```

Comment:
I think the package-lock.json should be committed into source repositories, see [here](https://docs.npmjs.com/files/package-lock.json).